.changeset/swift-gifts-film.md

```diff
@@ -0,0 +1,4 @@
+---
+---
+
+build(deps-dev): bump @graphql-codegen/typescript-generic-sdk from 3.1.0 to 4.0.1
```

Comment:
I suspected this could happen — the title seems too big. 👀

Reply:
@maschad I think the easiest path is just to rename the title/changeset manually and move on.

What do you think?

Reply:
Agreed, this is more of an edge case.

Reply:
[`a72bc54` (#2583)](https://github.com/FuelLabs/fuels-ts/pull/2583/commits/a72bc545257cdae1828585cfb87847de699d57c4)